ubclaunchpad/ubc-degree-navigator
# Fixed invalid sql type (slice) error in database

## Describe your changes
> Add a short description of the changes. These will be the things to check for when the reviewers review your PR.

Fixed the invalid sql type (slice) error for one-to-many relations in the database.

#### Screenshot (if applicable)
> This can be FE screenshot, DB screenshot, API call from network tab, etc.

## How do I test
> Provide breif steps on how to test these changes. 
1. Run go run . and make sure this error doesn't pop up in the terminal.
![image](https://github.com/ubclaunchpad/ubc-degree-navigator/assets/99298978/a65cd2cd-264f-4e6c-8579-e7c80717fc1f)


## Checklist before requesting a review
